policyengine_us/parameters/gov/hhs/medicaid/eligibility/categories/pregnant/income_limit.yaml

```diff
@@ -21,6 +21,7 @@ CA:
   2021-01-01: 2.13
 CO:
   2021-01-01: 2.00
+  2023-01-01: 1.95
```

Comment:
> Could you check the original Colorado source? I suspect KFF may have misinterpreted whether some states include the 5% disregard, given how many have changes of 0.05 here.

According to https://hcpf.colorado.gov/sites/hcpf/files/April%202024%20Medicaid%20Income%20Chart.pdf, it is actually 195% for pregnant women. But would this still be 200% then?

Reply:
Yes, I think 195% is the MAGI limit, where MAGI includes the 5% disregard as specified in https://www.law.cornell.edu/uscode/text/42/1396a#e_14_I_i. The [Colorado regs](https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11143) also indicate this.

As an example, [C.R.S. 25.5-8-103](https://advance.lexis.com/api/document/collection/statutes-legislation/id/653W-BRH3-CGX8-04B9-00008-00?cite=C.R.S.%2025.5-8-103&context=1000516) defines the income limit for pregnant women with children at 260%, which would be 265% after the disregard. I couldn't immediately find the statutory backing for 195% for pregnant women without children. Ideally we'd cite the legal code for all such numbers rather than the KFF source, but for now let's just double-check the ones KFF thinks changed by 5%.

Reply:
> for now let's just double-check the ones KFF thinks changed by 5%.

Ok, double checked it and fixed a few states according to their individual rules which are diff from KFF

Reply:
CO, NJ, and VA, is that right?

Reply:
> CO, NJ, and VA, is that right?

Yes